test/gtest/uct/ib/test_ib_md.cc

```diff
@@ -224,6 +226,78 @@ void test_ib_md::test_smkey_reg_atomic(void)
     ucs_mmap_free(buffer, size);
 }
 
+void
+test_ib_md::test_mkey_pack_mt_internal(unsigned access_mask, bool invalidate)
+{
+    constexpr size_t size = UCS_MBYTE;
+    unsigned pack_flags, dereg_flags;
+    void* buffer;
+    int ret;
+    uct_mem_h memh;
+
+    if ((access_mask & UCT_MD_MEM_ACCESS_REMOTE_ATOMIC) && is_bf_arm()) {
+        UCS_TEST_SKIP_R("FIXME: AMO reg key bug on BF device, skipping");
```

Comment:
what is the issue?

Reply:
I copied that message from `test_md::test_reg_mem` since it's doing the pretty similar thing. I didn't catch this error by myself.

Reply:
maybe it's not relevant anymore and can be removed? Can you try remove it and see whether CI passes.
@Artemy-Mellanox are you aware about this issue?